### ISSUE TITLE
server side error when file field inside group

### ISSUE THREAD

**Issue by pwkip** [pwkip/contact-form-7-conditional-fields#92]
there is a bug when a file field is inside a hidden group.

This will not work (throws 500 ERROR server side):
```
[text a]
[group g]
  [file file-2]
[/group]
[submit]
```

Temporary workaround for now:

Change file fields inside groups to required file fields, like this:

```
[text a]
[group g]
  [file* file-2]
[/group]
[submit]
```

Then it will work.

**Comment by pwkip** [comment]
Additional file error when file field inside nested repeater. 
- https://conditional-fields-cf7.bdwm.be/form-tester/?test-form=file+in+nested+repeater+%282%29
- https://conditional-fields-cf7.bdwm.be/form-tester/?test-form=file+in+nested+repeater+%281%29

**Comment by pwkip** [comment]
both issues fixed in version 2.2.6
